validation/jinja2/validation/_segment_card.html

```diff
@@ -56,41 +61,65 @@
+                        </td>
+                        {% if is_expert %}
+                        <td>
+                            {% if recording.quality == 'good' %}
+                                <button class="button button--success-solid audio-quality-good" data-cy="good-button" data-rec-id={{ recording.id }}>Good</button>
+                                <button class="button button--fail audio-quality-bad" data-cy="bad-button" data-rec-id={{ recording.id }}>Bad</button>
+                            {% elif recording.quality == 'bad' %}
+                                <button class="button button--success audio-quality-good" data-cy="good-button" data-rec-id={{ recording.id }}>Good</button>
+                                <button class="button button--fail-solid audio-quality-bad" data-cy="bad-button" data-rec-id={{ recording.id }}>Bad</button>
+                            {% else %}
+                                <button class="button button--success audio-quality-good" data-cy="good-button" data-rec-id={{ recording.id }}>Good</button>
+                                <button class="button button--fail audio-quality-bad" data-cy="bad-button" data-rec-id={{ recording.id }}>Bad</button>
+                            {% endif %} {# recording quality #}
+                        </td>
+                        {% endif %}
+                        {% if is_linguist %}
+                            <td>{{ recording.session.date }}</td>
+                        {% endif %}
+                    </tr>
+
+                {% endfor %}{# for recording in phrase.recordings #}
+            </table>
+            {% if is_linguist %}
+                <a href={{ url('validation:segment_detail', phrase.id) }} class="button button--success right"  data-cy="options-button">More Options</a>
+            {% endif %} {# is linguist #}
+        </div>
+
+        <!-- Modal -->
+        <div id="{{ phrase.id }}-modal" class="modal fade" role="dialog">
+            <div class="modal-dialog">
+
+                <!-- Modal content-->
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h4 class="modal-title">What's wrong with this entry?</h4>
+                    </div>
+
+                    <form action="" method="post">
+                        <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrf_token }}">
+                        <div class="modal-body">
+                            {{ forms[phrase.id] }}
+                        </div>
+                        <div class="modal-footer">
+                            <input data-cy="cancel-button" type="button" class="button button--fail" data-dismiss="modal" value="Cancel">
+                            <input data-cy="save-button" type="submit" class="button button--success" value="Save">
+                        </div>
+                    </form>
+                </div>
             </div>
-        </article>
+        </div>
```

Comment:
Consider using `<dialog>` instead: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

There is a way to mix this with `<form method="dialog">` to get some modal functionality "for free", but I don't know how to coax Django's generated `Form` classes with it.

Reply:
`<dialog>` is not supported on Firefox, IE, or Safari, so I don't actually think that's what I want to use.